triton-client/src/components/DatasetColumns.tsx

```diff
@@ -0,0 +1,371 @@
+                        content: `The total size of the datasets will exceed the ${downloadType} project quota. This dataset will be queued until space is freed.`,
+                        onOk: () =>
+                            dispatchCreateRequest(downloadType).catch((e) =>
+                                console.error(e),
+                            ),
+                        okText: "Continue",
+                        cancelText: "Cancel",
+                    })
+                } else {
+                    dispatchCreateRequest(downloadType).catch((e) =>
+                        console.error(e),
+                    )
+                }
+            }
+        },
+        [
+            constants.diskCapacity,
+            dataset,
+            dispatchCreateRequest,
+            project,
+            totalSize,
+        ],
+    )
+
+    if (
+        activeRequest !== undefined &&
+        currentRequest == activeRequest &&
+        !currentRequest.should_delete &&
+        currentRequest.status !== "FAILED"
+    ) {
+        const { type, status } = currentRequest
+        const actions: ActionDropdownProps["actions"] = [
+            {
+                action: {
+                    name: "Unstage dataset",
+                    actionCall: async () =>
+                        dispatch(deleteDownloadRequest(datasetID)).then(
+                            () => {
+                                notification.success({
+                                    message: "Dataset Unstaging",
+                                    description: `Dataset #${datasetID} will be unstaged shortly.`,
+                                })
+                                updateDataset(datasetID)
+                            },
+                            (e) => {
+                                notification.error({
+                                    message: "Error Unstaging Dataset",
+                                    description: `Dataset #${datasetID} could not be unstaged.`,
+                                })
+                                console.error(e)
+                            },
+                        ),
+                },
+                icon: <CloseCircleOutlined style={{ color: "#c9162b" }} />,
+            },
+            {
+                action: {
+                    name: "Extend staging",
+                    actionCall: async () => {
+                        await dispatch(extendStagingRequest(datasetID)).catch(
+                            (e) => console.error(e),
+                        )
+                        await updateDataset(datasetID)
+                    },
+                },
+                icon: <PlusCircleOutlined style={{ color: "#097969" }} />,
+            },
+        ]
+
+        let statusDescription: ReactNode
+        if (status === "SUCCESS") {
+            statusDescription = "DOWNLOAD"
+        } else {
+            statusDescription = "QUEUED"
+        }
```

Comment:
this section can be `{status === "SUCCESS" ? "DOWNLOAD": "QUEUED"}` instead of assigning the statusDescription variable and then using it as a string inside the component